bin/ssl.sh

```diff
@@ -0,0 +1,5 @@
+#! /bin/sh
+SSL_CERT="${SSL_CERT:-cert/astral.pem}"
```

Comment:
Should we call these "ASTRAL_SSL_CERT/KEY" just to distinguish them from the other env vars?

Reply:
good idea, will rename